submissions/kazeemErinfolami/html/images/contact.css

```diff
@@ -0,0 +1,543 @@
+* {
```

Comment:
contact.css appear twice. This one is the one inside the wrong folder, kindly remove it